### PR TITLE
Add default_browser outcome for Firefox iOS

### PR DESCRIPTION
**Pull request by scholtzan**
For https://docs.google.com/document/d/1UHKm_ko0zl_Q31aNwQwux2GK3-00IuVm55DJZs9mh1U/edit#

This needs someone from DS to take a look at the metric definitions.

### CODE REVIEW

File: outcomes/firefox_ios/default_browser.toml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+friendly_name = "Default browser"
+description = "Metrics that describe whether Firefox for iOS is a user's default browser."
+
+[metrics.default_browser_action]
```
Comment (scholtzan) [inline outcomes/firefox_ios/default_browser.toml:4]: I copied this metric from the fenix outcome: https://github.com/mozilla/jetstream-config/blob/08b4365aca5d57352e628a03111beb03122ead88/outcomes/fenix/default-browser.toml#L25

File: outcomes/firefox_ios/default_browser.toml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+data_source = "events"
+statistics = { binomial = {} }
+
+
+[metrics.opened_as_default]
+friendly_name = "Opened as default browser"
+description = """
+    Fraction of users who opened Firefox as default browser. 
+"""
+select_expression = "SUM(metrics.counter.app_opened_as_default_browser) > 0"
```
Comment (scholtzan) [inline outcomes/firefox_ios/default_browser.toml:22]: I'm not sure how we want to compute this metric. The document says something about 7 day retention which is currently not reflected here.

Comment (ksiegler1) [reply outcomes/firefox_ios/default_browser.toml:22]: Can we update the description for "Opened as default browser" to be "Fraction of users who opened Firefox through an external link, implying the client has Firefox set as the default browser"?

File: outcomes/firefox_ios/default_browser.toml (hunk 1 above)
Comment (ksiegler1) [reply outcomes/firefox_ios/default_browser.toml:4]: I don't believe we're able to track this (default_browser_action) for iOS. We have to use the proxy metric that measures whether they opened an external link through Firefox, which is our best indication of whether we are the default browser or not.

File: outcomes/firefox_ios/default_browser.toml (hunk 2 above)
Comment (scholtzan) [reply outcomes/firefox_ios/default_browser.toml:23]: Updated the description

File: outcomes/firefox_ios/default_browser.toml (hunk 1 above)
Comment (scholtzan) [reply outcomes/firefox_ios/default_browser.toml]: I've removed `default_browser_action` for now. We can always add more metrics later and update how we compute the metrics if we need to.

**Review by ksiegler1 — APPROVED** [review]

**Pull request merged**